Reimport css/filter-effect WPTs.

mattwoodrow opened the pull request:
#### 13249b9710ec9ca9c1f1a2afd29450c97491a6ec
<pre>
Reimport css/filter-effect WPTs.
<a href="https://bugs.webkit.org/show_bug.cgi?id=293993">https://bugs.webkit.org/show_bug.cgi?id=293993</a>

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/3de4cc4940f93be1ffe4fbf4889bec947e0e8d6a">https://github.com/web-platform-tests/wpt/commit/3de4cc4940f93be1ffe4fbf4889bec947e0e8d6a</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/filter-interpolation-sign-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/filter-interpolation-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-backdrop-filter-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-backdrop-filter.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-clip-path-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-clip-path-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-clip-path.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-filter-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-filter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-filter.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-mix-blend-mode-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-mix-blend-mode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-opacity-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-background-color-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-blur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-boundary-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-boundary.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-radius-zoom-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-radius-zoom-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-radius-zoom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect-zoom-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect-zoom-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect-zoom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-clipping-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-clipping-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-mirror-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-mirror.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-pixels-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-pixels-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-important-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-important.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-filter.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask-large-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask-large-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask-large.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-reference-filter-mutated-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-reference-filter-mutated.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-scale-transform-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-scale-transform-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-scale-transform.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-svg-blur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-svg-blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-transform-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-transform-popover-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-transform-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-transform.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/external-reference-in-interleaved-oof-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filter-transform-clip-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filter-transform-clip-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filter-transform-clip.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-brightness.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-opacity-hit-testing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-opacity-hit-testing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/drop-shadow-currentcolor-dynamic-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/drop-shadow-currentcolor-dynamic-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feComposite-intersection-feTile-input-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feComposite-intersection-feTile-input-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feComposite-intersection-feTile-input-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feComposite-intersection-feTile-input-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feComposite-intersection-feTile-input.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fecolormatrix-negative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fecolormatrix-negative-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fecolormatrix-negative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feimage-svg-image-with-nested-data-url-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-html-content-viewport.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-html-content-viewport.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-sign-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-subregion-01-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-subregion-01-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-subregion-01.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-drop-shadow-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-ref-brightness-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-ref-brightness-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-ref-brightness-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/hidpi-invert-filter-background-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/hidpi-invert-filter-background-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/hidpi-invert-filter-background.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/flood-opacity-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/flood-opacity-computed.svg:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/flood-opacity-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/flood-opacity-valid.svg:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-basic-blur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-boundary-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-clip-rect-2-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-edge-clipping-2-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-edge-mirror-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-edge-pixels-2-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-plus-will-change-opacity-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-svg-blur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filters-grayscale-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filters-grayscale-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filters-grayscale-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/background-image-blur-repaint-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/clip-under-filter-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/clip-under-filter-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/clip-under-filter-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/drop-shadow-clipped-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/dynamic-filter-changes-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-add-hw-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-after-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-delete-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-displacement-negative-scale-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-feimage-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-feimage-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-feimage-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-feimage-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-lighting-no-light.tentative-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-local-url-with-base-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-merge-no-inputs.tentative-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-obb-dimensions-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-on-span-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-on-transparent-element-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-rename-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-rename-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-source-alpha-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-source-alpha-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/empty-element-with-filter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/fecomposite-non-zero-inoffset-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-effect-remove-unattached-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-region-negative-positioned-child-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-region-transformed-child-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-region-transformed-composited-child-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-turbulence-invalid-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-url-to-non-existent-filter-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filters-drop-shadow-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filters-drop-shadow-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filters-drop-shadow-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filters-opacity-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filters-opacity-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/green-100x100.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/green-98-100x100.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/green-blue-stripe-100x100.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/reference-filter-update-on-attribute-change-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/remove-filter-repaint-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/repaint-added-backdrop-filter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/root-element-with-opacity-filter-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/svg-feflood-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/svg-feimage-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/svg-feoffset-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/svg-relative-urls-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/svg-relative-urls-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/resources/backdrop-filter-backdrop-root.css: Added.
(.container):
(.testcase):
(.testcase div):
(.testcase&gt;div):
(.testcase&gt;div&gt;div):
(*):
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/filtersubregion00.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/hueRotate.svg:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/simulate-backdrop-blur.js: Added.
(const.isAncestor):
(const.commonAncestor):
(const.computeOffset):
(const.cloneExcept):
(simulateBackdropBlur):
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/support/README: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/support/swatch-green.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/support/swatch-red.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/support/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/w3c-import.log:
* LayoutTests/platform/mac-wk2/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/295834@main">https://commits.webkit.org/295834@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/2d2c122e0dc2c30c1f41ff9be6de607f440cba5e

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/106294 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/26043 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/16439 "Built successfully") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/5/builds/111492 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/56890 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/130/builds/26706 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/123/builds/34546 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/5/builds/111492 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/56890 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/109298 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/130/builds/26706 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/95922 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/5/builds/111492 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/130/builds/26706 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/135/builds/14020 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe-cairo~~](https://ews-build.webkit.org/#/builders/65/builds/56330 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/130/builds/26706 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/136/builds/14055 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/2/builds/114352 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/33432 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/123/builds/34546 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/2/builds/114352 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/33796 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/92150 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/2/builds/114352 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 vision-wk2~~](https://ews-build.webkit.org/#/builders/126/builds/34393 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/12204 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/29016 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/17224 "Built successfully and passed tests") | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/127/builds/33357 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/125/builds/33103 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/129/builds/36456 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/124/builds/34701 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/filter-computed.html:
This undoes @weinig's changes from https://github.com/WebKit/WebKit/commit/5984529737faad47906ee14c18c1918f5f5e2f7c 

We should probably export them first

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/filter-parsing-valid.html:
ditto

nt1m reviewed (commented):
r=me modulo unexported change that got undone

mattwoodrow committed:
Reimport css/filter-effect WPTs.
https://bugs.webkit.org/show_bug.cgi?id=293993

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/3de4cc4940f93be1ffe4fbf4889bec947e0e8d6a

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/filter-interpolation-sign-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/filter-interpolation-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/animation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-backdrop-filter-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-backdrop-filter.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-clip-path-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-clip-path-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-clip-path.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-filter-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-filter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-filter.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-mix-blend-mode-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-mix-blend-mode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-opacity-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-backdrop-root-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-background-color-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-blur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-basic-blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-boundary-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-boundary.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-radius-zoom-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-radius-zoom-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-radius-zoom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect-zoom-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect-zoom-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-clip-rect-zoom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-clipping-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-clipping-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-mirror-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-mirror.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-pixels-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-edge-pixels-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-important-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-important.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-isolation.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-filter.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask-large-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask-large-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask-large.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-plus-mask.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-reference-filter-mutated-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-reference-filter-mutated.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-scale-transform-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-scale-transform-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-scale-transform.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-svg-blur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-svg-blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-transform-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-transform-popover-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-transform-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/backdrop-filter-transform.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/external-reference-in-interleaved-oof-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filter-transform-clip-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filter-transform-clip-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-backdrop-filter-transform-clip.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-brightness.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-opacity-hit-testing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/css-filters-opacity-hit-testing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/drop-shadow-currentcolor-dynamic-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/drop-shadow-currentcolor-dynamic-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/empty-element-with-filter-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feComposite-intersection-feTile-input-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feComposite-intersection-feTile-input-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feComposite-intersection-feTile-input-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feComposite-intersection-feTile-input-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feComposite-intersection-feTile-input.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fecolormatrix-negative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fecolormatrix-negative-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/fecolormatrix-negative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/feimage-svg-image-with-nested-data-url-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-html-content-viewport.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-region-html-content-viewport.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-sign-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-sign-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-subregion-01-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-subregion-01-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filter-subregion-01.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-drop-shadow-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-ref-brightness-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-ref-brightness-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/filters-ref-brightness-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/hidpi-invert-filter-background-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/hidpi-invert-filter-background-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/hidpi-invert-filter-background.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/flood-opacity-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/flood-opacity-computed.svg:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/flood-opacity-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/flood-opacity-valid.svg:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-basic-blur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-boundary-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-clip-rect-2-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-edge-clipping-2-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-edge-mirror-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-edge-pixels-2-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-plus-will-change-opacity-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-svg-blur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filter-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filters-grayscale-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filters-grayscale-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/backdrop-filters-grayscale-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/background-image-blur-repaint-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/clip-under-filter-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/clip-under-filter-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/clip-under-filter-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/drop-shadow-clipped-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/dynamic-filter-changes-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-add-hw-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-after-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-delete-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-displacement-negative-scale-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-feimage-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-feimage-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-feimage-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-feimage-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-lighting-no-light.tentative-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-local-url-with-base-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-merge-no-inputs.tentative-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-obb-dimensions-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-on-span-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-on-transparent-element-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-rename-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-rename-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-source-alpha-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/effect-reference-source-alpha-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/empty-element-with-filter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/fecomposite-non-zero-inoffset-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-effect-remove-unattached-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-region-negative-positioned-child-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-region-transformed-child-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-region-transformed-composited-child-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-turbulence-invalid-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filter-url-to-non-existent-filter-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filters-drop-shadow-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filters-drop-shadow-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filters-drop-shadow-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filters-opacity-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/filters-opacity-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/green-100x100.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/green-98-100x100.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/green-blue-stripe-100x100.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/reference-filter-update-on-attribute-change-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/remove-filter-repaint-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/repaint-added-backdrop-filter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/root-element-with-opacity-filter-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/svg-feflood-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/svg-feimage-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/svg-feoffset-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/svg-relative-urls-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/svg-relative-urls-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/resources/backdrop-filter-backdrop-root.css: Added.
(.container):
(.testcase):
(.testcase div):
(.testcase>div):
(.testcase>div>div):
(*):
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/filtersubregion00.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/hueRotate.svg:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/simulate-backdrop-blur.js: Added.
(const.isAncestor):
(const.commonAncestor):
(const.computeOffset):
(const.cloneExcept):
(simulateBackdropBlur):
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/support/README: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/support/swatch-green.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/support/swatch-red.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/support/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/svg-feimage-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/filter-effects/w3c-import.log:
* LayoutTests/platform/mac-wk2/TestExpectations:

Canonical link: https://commits.webkit.org/295834@main
